vmware-tanzu/community-edition
# Contour package: drop 1.15.1, add 1.17.2 and 1.18.1

## What this PR does / why we need it
Drops old Contour version: 1.15.1
Adds new Contour versions: 1.17.2, 1.18.1

## Details for the Release Notes (PLEASE PROVIDE)

```release-note
Drops old Contour version: 1.15.1
Adds new Contour versions: 1.17.2, 1.18.1
```

## Which issue(s) this PR fixes

Fixes: #1730

## Describe testing done for PR

- Run ytt manually and verify YAML is generated
- Apply YAML to cluster and ensure Contour starts up

## Special notes for your reviewer

Looking for some input on keeping older versions around - as a package owner I'd prefer to maintain fewer versions, but not sure if there are any TCE policies/standards here. Ideally we can at least drop 1.15.1 and 1.17.1 since there have been security-related patch releases for each of those minor versions.
